### PR TITLE
People Update HI

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update HI

### CODE REVIEW

**Comment by jessemortenson** [comment]
This is down to just one warning about a retired person with no active roles, which we are ignoring.

File: data/hi/legislature/Darius-K-Kila-76c9a0ff-a9f0-47ac-bf3d-9f8ee134c411.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+id: ocd-person/76c9a0ff-a9f0-47ac-bf3d-9f8ee134c411
+name: Darius K. Kila
+given_name: Darius K.
+family_name: Kila
+email: repkila@capitol.hawaii.gov
+image: http://www.capitol.hawaii.gov/MemberFiles/RepSenPhotos/Kila.jpg
+party:
+- name: Democratic
+roles:
+- type: lower
```
Comment (samanthacara) [inline data/hi/legislature/Darius-K-Kila-76c9a0ff-a9f0-47ac-bf3d-9f8ee134c411.yml:10]: Start dateL 10/8/2022

File: data/hi/municipalities/Derek-Kawakami-ff3fcb8b-c05c-44f6-95fd-454be5e05128.yml [hunk 2]
```diff
@@ -4,13 +4,13 @@ given_name: Derek
 family_name: Kawakami
 email: mayor@kauai.gov
 roles:
-- end_date: '2022-12-05'
+- end_date: '2026-12-05'
```
Comment (samanthacara) [inline data/hi/municipalities/Derek-Kawakami-ff3fcb8b-c05c-44f6-95fd-454be5e05128.yml:7]: David K. served from 2011-2016. End Date: 12.30.2016

File: data/hi/retired/Bob-McDermott-4f069ee2-9761-4de3-925a-48843627beed.yml [hunk 3]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/mcdermott.jpg
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Bob-McDermott-4f069ee2-9761-4de3-925a-48843627beed.yml:10]: end date: 11/30/2022

File: data/hi/retired/Brian-T-Taniguchi-364c43d8-5f71-49ca-b6a8-3df07088f00c.yml [hunk 4]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/taniguchi.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Brian-T-Taniguchi-364c43d8-5f71-49ca-b6a8-3df07088f00c.yml:10]: end date: 11/30/2022

File: data/hi/retired/Clarence-K-Nishihara-c8b1a434-cf69-4847-b6b3-22cc1f6dfbe6.yml [hunk 5]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/nishihara.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Clarence-K-Nishihara-c8b1a434-cf69-4847-b6b3-22cc1f6dfbe6.yml:10]: end date: 11/30/2022

File: data/hi/retired/Dale-T-Kobayashi-8f6b289a-f229-4921-948e-bc8952577315.yml [hunk 6]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/dkobayashi.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Dale-T-Kobayashi-8f6b289a-f229-4921-948e-bc8952577315.yml:10]: end date: 11/30/2022

File: data/hi/retired/Gil-Riviere-4ed98c6f-39ca-4ef4-b26b-fae37bf049db.yml [hunk 7]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/riviere.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Gil-Riviere-4ed98c6f-39ca-4ef4-b26b-fae37bf049db.yml:10]: end date: 11/8/2022

File: data/hi/retired/Laura-Acasio-e7fcd92a-bbb2-466d-bd6e-7fd18a544ac3.yml [hunk 8]
```diff
@@ -6,14 +6,10 @@ email: senacasio@capitol.hawaii.gov
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Laura-Acasio-e7fcd92a-bbb2-466d-bd6e-7fd18a544ac3.yml:9]: end date: 11/8/2022

File: data/hi/retired/Linda-Clark-633436ff-9164-4d8d-a207-4df91c53a1ae.yml [hunk 9]
```diff
@@ -7,14 +7,10 @@ party:
 - name: Democratic
 roles:
 - start_date: 2021-06-23
+  end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Linda-Clark-633436ff-9164-4d8d-a207-4df91c53a1ae.yml:10]: end date: 11/8/2022

File: data/hi/retired/Linda-Ichiyama-f263427a-454f-46b6-ba94-2e05fe82e802.yml [hunk 10]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/ichiyama.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Linda-Ichiyama-f263427a-454f-46b6-ba94-2e05fe82e802.yml:10]: end date: 11/8/2022

File: data/hi/retired/Lorraine-R-Inouye-b9c88ecc-477f-47dd-97a5-b90c7ec90696.yml [hunk 11]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/inouye.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Lorraine-R-Inouye-b9c88ecc-477f-47dd-97a5-b90c7ec90696.yml:10]: end date: 11/8/2022

File: data/hi/retired/Matthew-S-LoPresti-2fb131bd-5ac9-4a6b-8e11-a38af6bd1042.yml [hunk 12]
```diff
@@ -12,14 +12,10 @@ roles:
   jurisdiction: ocd-jurisdiction/country:us/state:hi/government
   district: '41'
 - start_date: 2020-11-03
+  end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Matthew-S-LoPresti-2fb131bd-5ac9-4a6b-8e11-a38af6bd1042.yml:15]: end date: 11/8/2022

File: data/hi/retired/Michael-Victorino-9d6194bf-3062-4fda-bd3f-2b4299539c16.yml [hunk 13]
```diff
@@ -7,9 +7,9 @@ roles:
 - end_date: '2023-01-02'
```
Comment (samanthacara) [inline data/hi/retired/Michael-Victorino-9d6194bf-3062-4fda-bd3f-2b4299539c16.yml:7]: end date: 11/8/2022

File: data/hi/retired/Mike-Gabbard-926b2548-5882-4163-9dcd-317ff923698d.yml [hunk 14]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/gabbard.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Mike-Gabbard-926b2548-5882-4163-9dcd-317ff923698d.yml:10]: end date: 11/8/2022

File: data/hi/retired/Patrick-Pihana-Branco-7a41b2a9-b21d-44f6-bd1d-075319959783.yml [hunk 15]
```diff
@@ -6,14 +6,10 @@ email: repbranco@Capitol.hawaii.gov
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Patrick-Pihana-Branco-7a41b2a9-b21d-44f6-bd1d-075319959783.yml:9]: end date: 11/8/2022

File: data/hi/retired/Nadine-K-Nakamura-4772b163-ea6d-4258-9142-7ba07c00fe29.yml [hunk 16]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/nakamura.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Nadine-K-Nakamura-4772b163-ea6d-4258-9142-7ba07c00fe29.yml:10]: end date: 11/8/2022

File: data/hi/retired/Rosalyn-H-Baker-61c787dc-dcc3-4f63-9f96-7279e2c3c62a.yml [hunk 17]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/baker.jpg
 party:
 - name: Democratic
 roles:
-- type: upper
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Rosalyn-H-Baker-61c787dc-dcc3-4f63-9f96-7279e2c3c62a.yml:10]: end date: 11/8/2022

File: data/hi/retired/Roy-M-Takumi-0c8111ac-9594-4299-8e4e-a4c012ef83ff.yml [hunk 18]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/takumi.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Roy-M-Takumi-0c8111ac-9594-4299-8e4e-a4c012ef83ff.yml:10]: end date: 11/8/2022

File: data/hi/retired/Scott-Y-Nishimoto-a2152aef-d03f-4050-8f98-1e31aabfedf1.yml [hunk 19]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/nishimoto.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Scott-Y-Nishimoto-a2152aef-d03f-4050-8f98-1e31aabfedf1.yml:10]: end date: 11/8/2022

File: data/hi/retired/Sharon-E-Har-ee51c11f-2098-4843-9504-558bbe02f521.yml [hunk 20]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/har.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Sharon-E-Har-ee51c11f-2098-4843-9504-558bbe02f521.yml:10]: end date: 11/8/2022

File: data/hi/retired/Stacelynn-KM-Eli-80fb08bd-6518-4109-bf2e-5915fe78bbb7.yml [hunk 21]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/eli.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Stacelynn-KM-Eli-80fb08bd-6518-4109-bf2e-5915fe78bbb7.yml:10]: end date: 11/8/2022

File: data/hi/retired/Sylvia-Luke-f3a50da6-6c22-4c5a-ba48-d10ee2188333.yml [hunk 22]
```diff
@@ -7,13 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/luke.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Sylvia-Luke-f3a50da6-6c22-4c5a-ba48-d10ee2188333.yml:10]: end date: 11/8/2022

File: data/hi/retired/Stacelynn-KM-Eli-80fb08bd-6518-4109-bf2e-5915fe78bbb7.yml (hunk 21 above)
Comment (bfossen-ce) [reply data/hi/retired/Stacelynn-KM-Eli-80fb08bd-6518-4109-bf2e-5915fe78bbb7.yml:10]: @samanthacara you can ignore HI for now these were fixed on another PR

File: data/hi/retired/Takashi-Ohno-b929718f-65eb-4f07-8ec6-97be39e3ad59.yml [hunk 23]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/ohno.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Takashi-Ohno-b929718f-65eb-4f07-8ec6-97be39e3ad59.yml:10]: end date: 11/8/2022

File: data/hi/retired/Tina-Wildberger-7f8720da-3e8c-42e2-84e4-565091a9da0e.yml [hunk 24]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/wildberger.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Tina-Wildberger-7f8720da-3e8c-42e2-84e4-565091a9da0e.yml:10]: end date: 11/8/2022

File: data/hi/retired/Troy-N-Hashimoto-3b99a717-698f-4bcf-a630-183bb63e623e.yml [hunk 25]
```diff
@@ -2,23 +2,26 @@ id: ocd-person/3b99a717-698f-4bcf-a630-183bb63e623e
...
 family_name: Hashimoto
-email: rephashimoto@Capitol.hawaii.gov
-image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/hashimoto.jpg
+email: rephashimoto@capitol.hawaii.gov
+image: http://www.capitol.hawaii.gov/MemberFiles/RepSenPhotos/Hashimoto.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Troy-N-Hashimoto-3b99a717-698f-4bcf-a630-183bb63e623e.yml:10]: end date: 11/8/2022

File: data/hi/retired/Ty-JK-Cullen-93f8af26-7b89-42b0-ad8d-a72e906aba29.yml [hunk 26]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/cullen.jpg
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Ty-JK-Cullen-93f8af26-7b89-42b0-ad8d-a72e906aba29.yml:10]: end date: 11/8/2022

File: data/hi/retired/Val-Okimoto-0893bbd0-5e0d-4dc4-aca4-cb771597301b.yml [hunk 27]
```diff
@@ -7,14 +7,10 @@ image: http://capitol.hawaii.gov/Members/Images/RepSenPhotos/okimoto.jpg
 party:
 - name: Republican
 roles:
-- type: lower
+- end_date: '2022-12-30'
```
Comment (samanthacara) [inline data/hi/retired/Val-Okimoto-0893bbd0-5e0d-4dc4-aca4-cb771597301b.yml:10]: end date: 11/8/2022

**Review by samanthacara — COMMENTED** [review]
Hawaii House of Representatives leave on election day if they are not re-elected. These are the majority of comments made on this people update.

**Pull request merged**